docs/build-apps/hello_world.md

```diff
@@ -375,7 +392,7 @@ Transaction [TXID] committed in round [COMMITTED_ROUND]
 
 # Wait for confirmation
 
-Successfully submitting your transaction to the network does not necessarily mean the network confirmed it. Always check that the network confirmed your transaction within a block before proceeding. 
+Successfully submitting your transaction to the network does not necessarily mean the network confirmed it. Always check that the network confirmed your transaction within a block before proceeding. Add utility functions.
```

Comment:
Remove this change. Looks like a note to self.

Reply:
Done